src/overview.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+store your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance on repeat-visits. You can use Workbox to store
+  and serve common resources locally, rather than from the network.
+
+The only difference between the two options is whether you store all or only
+some resources locally.
+
+## The technology behind Workbox
+
+Modern offline web apps are possible thanks to **service workers**. Service
```

Comment:
Link to https://developers.google.com/web/fundamentals/getting-started/primers/service-workers here?